EmerisHQ/demeris
# fix: changed single airdrop implementation to a filter system instead of uing the store to fix refresh on single airdrop page issue

## Description
Initially, on click of a single airdrop on the airdrops table, i was setting the current selected airdrop in the store and retrieving that current selected airdrop from whichever component that needs it. This created an issue whenever the app is refreshed as the current selected airdrop object is lost because of the non persisting state of the store.

So i changed the implementation such that on load of the single airdrop page, i get all airdrops (loaded from App.vue file whenever the app launches and feature flag is turned on) and filter by the token ticker present on the active url (passed whenever an airdrop is clicked on the airdrops table), screenshot below:
<img width="497" alt="Screenshot 2022-04-19 at 5 01 56 PM" src="https://user-images.githubusercontent.com/49952972/164046551-32f095bb-a7ad-4f2e-9881-cd16cf202a0c.png">


## Feature flags
Please use this feature flag for testing ?VITE_FEATURE_AIRDROPS_FEATURE=1

## Testing

1. Go to app
2. Add feature flag
3. Go to airdrops page
4. Click on an airdrop to go to the single airdrop page
5. Refresh the page and see airdrop data not lost and page loads properly
